lib/admin/components/ProjectAccessSettings.js

```diff
@@ -57,27 +58,16 @@ export default class ProjectAccessSettings extends Component<Props, State> {
     this.props.projectAccessUpdated(this.props.project.id, access)
   }
 
-  feedsUpdated = () => {
+  feedsUpdated = (evt: SyntheticInputEvent<HTMLInputElement>) => {
```

Comment:
Are you OK with me just fixing the name of this handler method instead of adding JS doc? It should be `onToggleFeedSource` -- that may be where some of the confusion is coming from.